### PR TITLE
Add register app

### PR DESCRIPTION
**Pull request by pablo-grande**
To test:
1. Register a token with django shell
```
from models import Token
t = Token()
t.save()
```
2. Get token code and go to the register URL with code as parameter `localhost:<port>/register/<code>/`
3. Try the form
4. Try accessing same URL with invalid token code

### CODE REVIEW

File: register/views.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+def register(request, token=None):
+    if request.method == 'POST':
+        user_form = UserCreationForm(request.POST, prefix='user_form')
+        entity_form = EntityForm(request.POST, prefix='entity_form')
+        if user_form.is_valid() and entity_form.is_valid():
+            username = user_form.save()
+            entity = entity_form.save(commit=False)
+            entity.user = User.objects.get(username=username)
+            entity.save()
+            return redirect('/')
```
Comment (Nosk14) [inline register/views.py:17]: I think it would be nice to redirect the user to the backoffice login page

File: register/models.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+class Token(models.Model):
+    code = models.CharField(max_length=43)
+    expiry = models.DateTimeField()
+
+    def save(self, *args, **kwargs):
+        if not self.pk:
+            self.code = token_urlsafe()
+            self.expiry = timezone.now() + timezone.timedelta(days=1)
```
Comment (Nosk14) [inline register/models.py:13]: I don't feel comfortable with these two lines. I mean, when you create a new token like `t = Token()` you are creating an empty object and the valuers are set when you save it. I think the values should be set on the constructor. 
We can talk about it

**Review by Nosk14 — CHANGES_REQUESTED** [review]
The same token can be used to register a new entity as many times as the user wants. We should add a new field to know if it is already used or maybe we can just set the expiry time to epoch or something like that to invalidate it.

**Comment by pablo-grande** [comment]
Migrating PR to new branch feature/register

**Pull request closed without merging**